config/jobs/kubernetes/sig-release/cip/container-image-promoter.yaml

```diff
@@ -88,11 +88,11 @@ presubmits:
         - files
         - --manifests=/home/prow/go/src/github.com/kubernetes/k8s.io/artifacts/
         - --dry-run=true
-  kubernetes-sigs/k8s-container-image-promoter:
+  kubernetes-sigs/promo-tools:
```

Comment:
Rename this file/path?

Reply:
@spiffxp -- Done! Rebased and renamed to `config/jobs/kubernetes/sig-release/promo-tools/promo-tools.yaml`.